backend/controllers/activityList.js

```diff
@@ -1,26 +1,42 @@
+
 const express = require('express');
 const router = express.Router();
 const elite2Api = require('../elite2Api');
 const asyncMiddleware = require('../middleware/asyncHandler');
+const switchDateFormat = require('../utils');
 
 router.get('/', asyncMiddleware(async (req, res) => {
   const viewModel = await getActivityList(req, res);
   res.json(viewModel);
 }));
 
-const getActivityList = (async (req, res) => {
-  req.query.usage = 'PROD';
-  const activities = await elite2Api.getActivityList(req, res).data;
+const getActivityList = async (req, res) => {
+  // once request is not used passed to downstream services we wont have to manipulate it in this dodgy way
+  switchDateFormat(req);
```

Comment:
needs tidying

Reply:
Why not add start splitting request parameters out now? elite2api.getActivities(req,red,date) should be fairly straight forward